Content.Server/PowerCell/PowerCellSystem.Draw.cs

```diff
@@ -14,11 +14,11 @@ public sealed partial class PowerCellSystem
     public override void Update(float frameTime)
     {
         base.Update(frameTime);
-        var query = EntityQueryEnumerator<PowerCellDrawComponent, PowerCellSlotComponent, ItemToggleComponent>();
+        var query = EntityQueryEnumerator<PowerCellDrawComponent, PowerCellSlotComponent>();
 
-        while (query.MoveNext(out var uid, out var comp, out var slot, out var toggle))
+        while (query.MoveNext(out var uid, out var comp, out var slot))
         {
-            if (!comp.Enabled || !toggle.Activated)
+            if (!comp.Enabled || !Toggle.IsActivated(uid))
```

Comment:
This is still coupled to itemtoggle when powercelldraw is supposed to be generic.

Reply:
the resolve uses false now so it isnt required

Reply:
actually removed item toggle from the power cell code now it's its own thing